MIGRATING-0.6-0.7.md

```diff
@@ -396,4 +549,14 @@ Note that the table above shows the new fonts' `ascii` variants only. Some new f
 
 To style fonts, use the `MonoTextStyle` struct. `TextStyle` still exists, but has been repurposed to provide a more general interface to text styling. This more closely mirrors the way primitives are built and styled.
 
-TODO: Discuss baseline font alignment, etc, as per GH comment.
+The default vertical alignment for fonts is now the font's baseline. To retain the 0.6 behaviour and position text using its top-left corner, set the `baseline` property to `Baseline::Top`:
```

Comment:
We should use "baseline" instead of "vertical alignment" to be consistent with the other docs.

Reply:
"The default baseline for fonts is now the font's baseline" is strange to read here though. I'd be ok keeping it as "vertical alignment" as that describes the behaviour as opposed to the property. Can you think of a better way of bringing this across?

Reply:
Just bumping this in case you missed it :)